docs/shocks.rst

```diff
@@ -1,140 +1,134 @@
-Tutorial: Collisionless shocks
+.. default-role:: math
+
+
+Tutorial: Collisionless Shocks
 ##############################
 
-In this tutorial we use Runko to run a small non-magnetized collisionless shock simulation.
+This tutorial provides some tips for getting started with collisionless shock simulations using Runko.  
+
+Simulation scripts and analysis files for shock simulations are found in    
+
+.. code-block:: bash
+
+    /runko/projects/pic-shocks
 
-.. image:: https://cdn.jsdelivr.net/gh/natj/pb-utilities@master/movies/shock.gif
+The *pic-shocks* project simulates a scenario where plasma travels in a tube-like structure in the `-x` direction, meets the end of the tube and reflects back towards the `+x` direction. As a result, there are two populations of plasma passing through each other. The populations interact with each other via electromagnetic interactions due to the low-density nature of the plasma, forming a collisionless shock.
```

Comment:
I suggest a slight rewrite of the last sentences (cosmetics only) and specifying the shock type in another paragraph. 

As a result, there are two populations of plasma passing through each other. The populations interact with each other via the electromagnetic fields and form a collisionless shock. 

The example setup simulates a relativistic, magnetized, perpendicular shock in an electron-positron pair plasma. Such shocks are prone to the synchrotron maser instability. More details are available, e.g., in [Plotnikov & Sironi 2019](https://academic.oup.com/mnras/article/485/3/3816/5370092).